ztoc/fbs/ztoc.fbs

```diff
@@ -22,7 +22,7 @@ table FileMetadata {
 	devmajor : long;		// Major device number (valid for TypeChar or TypeBlock)
 	devminor : long;		// Minor device number (valid for TypeChar or TypeBlock)
 
-	xattrs : [Xattr];
+	xattrs : [Xattr];       // Raw PAXRecords from the tar file. The name is wrong, but changing it is backwards incompatible
```

Comment:
Should we ever make a major version change, would we want to look into correcting this?

Reply:
Added a note to https://github.com/awslabs/soci-snapshotter/issues/494. We need to think more about ztoc versioning in general.